Mobile nav updated

akashvenus opened the pull request:
## Changes proposed

<!-- List all the proposed changes in your PR -->

- Mobile view nav bar spacing updated

<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
<!--
[x] - Correct; marked as done
[X] - Correct; marked as done
[ ] - Not correct; marked as **not** done
-->

## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

## Screenshots

<!-- Add all the screenshots which support your changes -->

![Screenshot (89)](https://github.com/EddieHubCommunity/good-first-issue-finder/assets/59107567/6a0beabb-c873-40e3-8cd6-aff077f5333b)

Cahllagerfeld commented on package-lock.json:
Could you please revert these changes

akashvenus replied:
Yes I can. Should I reduce the padding? I included it incase the person touches an cm to the side of the content.

Cahllagerfeld commented on src/lib/components/hamburger-menu/menu.svelte:
```diff
@@ -48,7 +48,7 @@
         <ul>
           {#each navItems as item}
-            <li>
+            <li class="mt-9 px-3">
```
I feel this margin plus a padding is a bit too large what do you think?

akashvenus replied:
Maybe I can reduce the padding. Just wanted to ask you regarding this, since there only two options in the menu

akashvenus commented:
Hi there @Cahllagerfeld I've reverted the package-lock.json changes. Idk why it removed 5000 + lines. Also, I've fixed the margin spacing. 

![image](https://github.com/EddieHubCommunity/good-first-issue-finder/assets/59107567/98356741-98b5-4367-a3c1-bd46a187ed63)

akashvenus commented:
@Cahllagerfeld could you please check the updates

Cahllagerfeld commented on src/lib/components/hamburger-menu/menu.svelte:
```diff
@@ -48,7 +48,7 @@
         <ul>
           {#each navItems as item}
-            <li>
+            <li class="mt-6 px-3">
```
Instead of having `mt-6` applied here, you can also try using `space-y-6` or `space-y-4` on the `<ul>` element. This is a bit cleaner in my opinion

akashvenus replied:
Seems right. Will make the changes

Cahllagerfeld commented on src/lib/components/hamburger-menu/menu.svelte:
```diff
@@ -61,7 +61,7 @@
           <a
             data-test-id="login-btn"
-            class="default-transition rounded-xl bg-skin-text px-4 py-2 text-skin-off-background hover:bg-primary-100 dark:bg-skin-text-highlight dark:text-skin-background dark:hover:bg-primary-100"
+            class="default-transition mt-5 rounded-xl bg-skin-text px-4 py-2 text-skin-off-background hover:bg-primary-100 dark:bg-skin-text-highlight dark:text-skin-background dark:hover:bg-primary-100"
```
I think we should be consistent with margins and use the same number here as above

akashvenus replied:
Got it. Will change the following

Cahllagerfeld commented on package-lock.json:
The package-lock shouldn't be touched in that PR, could you please revert it 🙏

akashvenus replied:
I did revert it. Didn't it change?

akashvenus commented:
@Cahllagerfeld Could you please check the changes

Cahllagerfeld reviewed (approved):
Lets merge it!
